src/GoTrueClient.ts

```diff
@@ -343,6 +343,7 @@ export default class GoTrueClient {
       redirectTo: credentials.options?.redirectTo,
       scopes: credentials.options?.scopes,
       queryParams: credentials.options?.queryParams,
+      skipBrowserRedirect: credentials.options?.skipBrowserRedirect,
```

Comment:
I think this might be slightly confusing for some because some might interpret the "redirect" here as the redirect after the oauth flow has completed when we're actually referring to the redirect to the oauth provider's page.

Reply:
Yeah let's merge this next year (hehe) to see if I can come up with a better name.

Reply:
I assume the objective here is to eventually sign the user in via oauth, but not yet? And that the dev is now responsible for triggering the oauth provider url, which is returned.

Other possibilities:
`delayProviderSignIn`
`enableMiddleware` - if that's why this option is being introduced
`autoAuthorization` - would need a default of `true`

Reply:
> I assume the objective here is to eventually sign the user in via oauth, but not yet?

Yup, the purpose is to return the oauth provider's URL so the developer can decide how they want to redirect the user to the site. I'd also prefer for the naming to be more explicit such that it encompasses the meaning of the oauth provider URL being returned.

Reply:
Thanks for the clarification.

>I'd also prefer for the naming to be more explicit such that it encompasses the meaning of the oauth provider URL being returned.

I'm having trouble coming up with verbiage to express this - especially since the URL is returned no matter what.